scripts/tests/test_validate_min_versions_in_sync.py

```diff
@@ -10,34 +10,36 @@
     pin_min_versions_to_yaml_file,
 )
 
+BASE_PATH = pathlib.Path(__file__).parents[2]
```

Comment:
Might be more robust to define `pathlib.Path("scripts/tests/data/").resolve()` that can be used for the test below